src/lib/components/submit/SubmitAddNodeProvider.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+	import Input from '$lib/components/ui/Input.svelte';
+
+	let title = '';
```

Comment:
`title` and `summary` are named this way because it's a work in progress or it's the definitive naming?

if the first, can you add a TODO, if the latest, is is accurate? I'm not sure why for example the variable `title` is used for a "register a node provider".

Reply:
<img width="697" alt="image" src="https://github.com/peterpeterparker/proposals.network/assets/79041287/9b74da16-4568-488d-a189-e16c501c2751">
[source](https://wiki.internetcomputer.org/wiki/Node_Provider_Onboarding#8._Register_your_Node_Provider_principal_to_the_network)

Hi @peterpeterparker, 
I couldn't find the exact metadata field names in the .did files so I based it off what was shown in the icadmin command in the node provider onboarding runbook (see image and link above). I do agree that better naming could be used, but I wanted to stay as close to what is already documented. WDYT?

Reply:
I see, thanks for the explanation. We should stay close to the API and Candid declarations code wise. Those types are exposed by `@dfinity/nns` and other libraries.